[Sheeeesh-code] IP

Sheeeesh-code opened the pull request.

Commit:
Rename Duke to Amadeus and add skeleton

Commit:
Add echo command and d-mail command (an echo variant with different dialogues)

Commit:
Add List command

Commit:
Add Unmark, mark command and Task class

LJQ2001 commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -43,6 +43,7 @@ public static void main(String[] args) {
                 break;
             }else if(answer.equals("Echo")){
                 System.out.println("The Mad Scientist chose the option Echo");
```
Probably add comments on what each mode means like D-mail or El Psy Kongroo

LJQ2001 commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -62,6 +63,7 @@ public static void main(String[] args) {
             else if (answer.equals("D-mail")) {
                 System.out.println("The Mad Scientist chose to send a D-mail");
+                System.out.println("D-mail mode activated. Type 'El Psy Kongroo' to exit.");
                 while (true) {
```
Great Job in avoiding deep nesting for your ifs

LJQ2001 commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -91,9 +94,23 @@ else if (answer.equals("List")) {
                         for (int i = 0; i < count; i++) {
                             System.out.println((i + 1) + ". " + tasks[i]);
                         }
+                    } else if (echo.toLowerCase().startsWith("mark")) {
```
Code is structed Logically in a neat orientation (Like each printLn is easily readable on each line)

LJQ2001 commented on src/main/java/amadeus/Task.java:
```diff
@@ -0,0 +1,24 @@
+package amadeus;
```
There is an extra Task.class file in the commit. Remove it if not needed.

LJQ2001 reviewed (commented):
Good Job !!!

abefqy commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -0,0 +1,129 @@
+        System.out.println("What knowledge do you seek today?");
+        System.out.println();
+        System.out.println("Here are the different mode : 1. Echo");
+        System.out.println("                              2. D-mail");
```
Maybe rename D-mail to a more intuitive term or provide more guidance to user on the function

abefqy commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -0,0 +1,129 @@
+                            System.out.println("────────────────────────────────────────────────────────────────");
+                        } else {
+                            System.out.println("────────────────────────────────────────────────────────────────");
+                            System.out.println("You already have too many things to do, finish some before adding more.");
```
Good error handling by highlighting to the user

abefqy commented on src/main/java/amadeus/Task.class:
Remove redundant class file?

abefqy commented on src/main/java/amadeus/Amadeus.java:
```diff
@@ -0,0 +1,129 @@
+
+public class Amadeus {
+    public static void main(String[] args) {
+        Task[] tasks = new Task[100];
```
Perhaps you can change the size of the array to a variable called maxTask and initialised as 100, then subsequent changes to the size of the array can be edited just by changing maxTask to the new desired value.

abefqy reviewed (commented):
Good job! Very unique personalisation of your chatbot!

Commit:
Add clas Todos, Deadlines, and Events using inheritance

Commit:
Add test files (input file, expected output, actual output)

Commit:
Add Exception Class for handle error, and modify Amadeus to handle unknown command and todo without description

Commit:
Merge branch-Level-5 into master

Commit:
Use of ArrayList instead of array for task and implement of the command delete

Commit:
Added level 7 save implementation

Commit:
Corrected Amadeus

Commit:
Merge branch-Level-6

Commit:
Merged branch

Commit:
Corrected some bugs due to merge

Commit:
Adjusted to Git-Standard

Commit:
Improved code readabilty, and reorganisation of it

Commit:
more OOP

Commit:
Added Times ands Dates

Commit:
Implemented Find command

Commit:
Implemented JavaDoc

Sheeeesh-code committed:
Added UserGuide

Updated the user guide to reflect the Amadeus chatbot features and usage instructions.

Sheeeesh-code committed:
Add files via upload

add image

Sheeeesh-code committed:
Change product screenshot to image_amadeus.png

Updated product screenshot in README.

Sheeeesh-code committed:
Update README.md

Sheeeesh-code committed:
Update D-mail mode explanation in README

Clarified D-mail mode description in README.

Sheeeesh-code committed:
Update README.md

Sheeeesh-code committed:
Rectified UserGuide
